setup.py

```diff
@@ -16,6 +18,9 @@
 with open('README') as readme_stream:
     readme = readme_stream.read()
 
+SETUPTOOLS_PY3_SUPPORT = "setuptools<=57.5.0"
```

Comment:
Latest version still supporting this import: `from distutils.command.build_py import build_py_2to3 as build_py`